Sources/GDSAnalytics/README.md

```diff
@@ -150,6 +150,56 @@ let event = LinkEvent(textKey: "Confirm your identity another way",
 | link domain | The web domain of the link that was pressed. | Lowercase. Limited to 100 characters. | `https://signin.build.account.gov.uk` |
 | external | Indicates whether the link is to an external domain. | Lowercase. | `true` or `false` |
 
+### Popup Alerts
+
+This type of event should be logged when a call-to-action is displayed, usually this would be a `UIAlertController` but it may also be used for other modal inputs. Use the `PopupEvent` type for this.
+
+```swift
+let event = PopupEvent(textKey: "Change your permissions")
+```
+
+#### Expected Parameters:
+
+| Parameter | Description | Expected Format | Example |
+| ------------- |-------------| -----| --- |
+| name | The name of the event that occured, for popups this is `popup`. | N/A | `popup` |
+| type | The type of event that occured, for popups this is `call to action`. | N/A | `call to action` |
+| text | The text that is displayed on the popup. | Lowercase. Limited to 100 characters. | `change your permissions` |
+
+### Action Menu 
+
+This type of event fires when a custom systemAlert popup appears on a the screen, usually this would be a `UIAlertController` of the type `.actionSheet`. Use the `PopupEvent` type for this.
```

Comment:
"on a the" might need a correction
Also, should a `PopupEvent` be used for this? 🙂

Reply:
Good catch, thanks!

Reply:
updated now!